### PR TITLE
Release 1.16.0-RC2

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: debian/jessie/foreman-installer/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+foreman-installer (1.16.0-RC2-1) stable; urgency=low
```
Comment (mmoll) [inline debian/jessie/foreman-installer/changelog]: this needs to be `1.16.0~rc2-1`

**Comment by mmoll** [comment]
the branch name probably needs to start with `deb/1.16`

File: debian/jessie/foreman-installer/changelog [hunk 2]
```diff
@@ -1,3 +1,9 @@
+foreman-installer (1.16.0~RC2-1) stable; urgency=low
```
Comment (mmoll) [inline debian/jessie/foreman-installer/changelog]: RC2 -> rc2 (lowercase)

Comment (ekohl) [reply debian/jessie/foreman-installer/changelog]: Updated

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**